### DISCUSSION TITLE
Midi Create with Python 8.5 (GUI tkinter) and Mido

### DISCUSSION THREAD

**Discussion by dado57** [mido/mido#487] [Show and tell]
This program creates midi files starting from pre-existing patterns with parameters like start key note, scale.
it is also possible to make changes to the patterns. Play the midi and save it.
This is the link on github
(https://github.com/mido/mido)
![DadoCrea](https://github.com/mido/mido/assets/70191990/ef960c6c-abef-4a34-8146-d78c36b05c72)

Thanks to Ole Martin Bjørndalen and everyone who contributed.
I'm Italian and my English is translated with Google.
I would be very happy for your comment.

**Comment by rdoursenaud** [comment]
Great job, looks nice!
I'll make sure to try it.
Thanks for sharing.

By the way, it seems that you inadvertently inserted mido's URL instead of your project's: https://github.com/dado57/MidiCrea

**Reply by dado57** [reply]
True, thanks Raphaël for reporting.
Now I can share the youtube link as well: [https://youtu.be/pyVhJaYHakM](https://youtu.be/pyVhJaYHakM)
Voice in Italian, subtitles (a little playful in English) ... always translated with google.

**Reply by rdoursenaud** [reply]
Thanks for sharing!
